Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -141,6 +142,25 @@ public void RoslynPlugin_GenerateForMultiLevelAnalyzers_Succeeds()
             AssertJarsGenerated(outputDir, 3);
         }
 
+        [TestMethod]
+        public void RoslynPlugin_ThirdPartyAnalyzer_1_Succeeds()
+        {
+            // Note: this test will access the public nuget site if the package
+            // can't be found locally so:
+            // 1) it can be slow and
+            // 2) it it more likely to fail due to environmental issues
+            CheckCanGenerateForThirdPartyAssembly("wintellect.analyzers", new SemanticVersion("1.0.6"));
+        }
+
+        [TestMethod]
+        public void RoslynPlugin_ThirdPartyAnalyzer_2_Succeeds()
+        {
+            // Build against the latest public version (currently v5.6)
+            // Note: this test could fail if the third-party analyzer is updated
+            // to use a newer version of Roslyn than the one supported by the SDK
+            CheckCanGenerateForThirdPartyAssembly("RefactoringEssentials", null /* latest */);
```

Comment:
Two new integration tests against publid third-parties analyzers that are built against different versions of Roslyn.